doc/docs/deployment/digital-ocean-apps.md

```diff
@@ -0,0 +1,185 @@
+
+After application is created, you'll land on application dashboard page. Here you can see application status, check runtime logs, track deployment status, and manage application settings.
+
+Before proceed to next steps, navigate to settings and expand `App spec` tab. Make sure it contains `dockerfile_path` key with correct path to application's Dockerfile.
+
+![Create app review](/img/deployment/digital-ocean-apps/do-settings-app-spec.png)
+
+## Mongo Atlas
+
+Navigate to [mongodb](https://www.mongodb.com/cloud/atlas/register) and sign in into your account and create new database
+
+Step-by-Step database creation guide:
+
+1. Select desired type (`Dedicated` for production environment or `shared` for sandbox)
+2. Select desired provider and region(recommended to select same or close region to DO application region)
+3. Select cluster tier (`M0 Sanbox` for sanbox environments, for production it is recommended to selec one of the paid tiers)
+4. [**Optional**] Change cluster name
+
+![Mongo cluster](/img/deployment/digital-ocean-apps/mongo-create.png)
+
+After cluster is created, You'll need to setup security
+1. Select authentication type (username and password)
+
+![Mongo setup authentication](/img/deployment/digital-ocean-apps/mongo-create-password.png)
+
+2. Add list of ip addresses, which should have access to your cluster. (Add `0.0.0.0/0` ip address to allow anyone with credentials connect to cluster)
+
+![Mongo setup ip white list](/img/deployment/digital-ocean-apps/mongo-create-ip-list.png)
+
+Once database created, you'll land on dashboard page. Here you can get URI connection string by pressing `connect` button.
+
+![Mongo dashboard](/img/deployment/digital-ocean-apps/mongo-dashboard.png)
+
+Select `Connect your application` option. Chose driver and mongo version, and copy connection string. Don't forget ro replace `<password>` with password, added in step 1
+
+![Mongo connect dialog](/img/deployment/digital-ocean-apps/mongo-connection-string.png)
+
+Navigate to Digital Ocean App and add connection string to application's environment variables (under `settings` tab)
+
+![Digital Ocean App's environment variables](/img/deployment/digital-ocean-apps/mongo-add-variable.png)
+
+## Redis
+
+Navigate to [redis](https://redis.com/try-free/) and create an account.
+
+Select cloud provider and region and press `Let's start free` to finish database creation.
+
+![Redis create database](/img/deployment/digital-ocean-apps/redis-creation.png)
+
+Open settings of created database by clicking on it.
+
+On the settings screen you can find public endpoint of database aswell as its password
+
+![Redis public endpoint](/img/deployment/digital-ocean-apps/redis-public-endpoint.png)
+![Redis password](/img/deployment/digital-ocean-apps/redis-password.png)
+
+Form redis connection string using public endpoint and password - `redis://:<password@<public-endpoint>` and add this to DO Application environment variables
```

Comment:
```suggestion
Form Redis connection string using public endpoint and password `redis://:<password@<public-endpoint>`. Save this value. It will be needed later when creating the app in Digital Ocean.
```